humanguide/other-design-considerations/scaling-images.rst

```diff
@@ -0,0 +1,27 @@
+Scaling images
+==============
+
+Scaling images is important in a convergent app. App icons and images can be created in scalable formats to improve their ability to scale.
+
+.. figure:: /_static/images/humanguide/53.png
+   :align: center
+
+One of the most popular scalable formats is `SVG <https://en.wikipedia.org/wiki/Scalable_Vector_Graphics>`__. SVG files can be created using software like `Inkscape <https://inkscape.org/>`__.
+
+One particularity of qml, though, is that svg are rendered to use low memory and losing the advatadges of scalability. When using a SVG file in an ``Image`` element we can use the property called ``sourceSize``. This will define the rendered size of the SVG image. A SVG image with a ``sourceSize`` width and height of 252 pixels, will be rendered as if the scalable image would be a png of 252 per 252 pixels.
```

Comment:
It may be better to say, "SVGs are rendered to use less memory, losing the advantages of scalability." It may also help to link [Performance tip Images on the Qt wiki](https://wiki.qt.io/Performance_tip_Images).

Do you mean "peculiarity" rather than "particularity"?

Reply:
Mmm... I would understand that peculiar is that is a kind of weird behaviour, but I want to express that this is a «particular» (or specific) feature that qml has.
Does it has another meaning in English?

Reply:
It seems technically correct, but it's not a word I've seen used very often.

Reply:
Ok :)